storage/perfschema/table_session_variables.cc

```diff
@@ -91,7 +91,7 @@ PFS_engine_table *table_session_variables::create(PFS_engine_table_share *) {
 ha_rows table_session_variables::get_row_count(void) {
   mysql_mutex_lock(&LOCK_plugin_delete);
 #ifndef NDEBUG
-  extern mysql_mutex_t LOCK_plugin;
+  extern mysql_mutex_t LOCK_plugin [[maybe_unused]];
```

Comment:
PS code does not have [[maybe_unused]] attribute. Why do we need it for PXC?
It should be in WITH_WSREP block.